_layouts/home.html

```diff
@@ -1,15 +1,8 @@
 ---
 layout: default
 ---
-
 <div class="home">
```

Comment:
No objections to this, but this just wraps the home page in a `div` of class `home`? Are we using that for something? If not, we could probably delete `home.html` and just use the `default` layout.

Reply:
Removed home layout